assets/src/scripts/components/graph/view/construction.js

```diff
@@ -25,6 +27,10 @@ const drawElement = function (store, elem, opts, element, index) {
                 .call(callIf(element.type === 'screen', (rect) => {
                     rect.attr('fill', `url(#screen-pattern-${index % 2})`);
                 }))
+                .call(callIf(element.type === 'borehole', (rect) => {
+                    rect.attr('fill', '#995500')
+                        .attr('opacity', '0.25');
+                }))
```

Comment:
Following the pattern above to render certain types. In the future it might be nice to put this behavior in the opts.

Reply:
yes, I agree. I knew I was creating a pattern here but was not sure yet where to place it. I will see if I can refactor this now.